routing/pathfind_test.go

```diff
@@ -1443,6 +1443,87 @@ func TestRouteFailMinHTLC(t *testing.T) {
 	}
 }
 
+// TestRouteFailMaxHTLC tests that if we attempt to route an HTLC which is
+// larger than the advertised max HTLC of an edge, then path finding fails.
+func TestRouteFailMaxHTLC(t *testing.T) {
+	t.Parallel()
+
+	graph, err := parseTestGraph(basicGraphFilePath)
```

Comment:
I believe we now use `symmetricTestChannel` for new pathfinding tests added to set up  a test graph rather than relying on the test graph file. You can take a look at some of the other tests for examples on how to use it.

Reply:
fixed